gitlab/resource_gitlab_project_access_token.go

```diff
@@ -0,0 +1,199 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectAccessToken() *schema.Resource {
+	// lintignore: XR002 // TODO: Resolve this tfproviderlint issue
+	return &schema.Resource{
+		Create: resourceGitlabProjectAccessTokenCreate,
+		Read:   resourceGitlabProjectAccessTokenRead,
+		Delete: resourceGitlabProjectAccessTokenDelete,
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeInt,
```

Comment:
The `project` field is of type `schema.TypeString` for (almost) all other resources. I highly recommend to change it from an int to string here, too.